lib/engine/step/minor_half_pay.rb

```diff
@@ -5,12 +5,13 @@ module Step
     module MinorHalfPay
       def actions(entity)
         return [] if entity.minor?
+        return [] if entity.corporation? && entity.type == :minor
```

Comment:
Allow minor's that are based on the corporation class to also take advantage of the module.